src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public String toString() {
+        return String.format("[%s] %s", getStatusIcon(), this.description);
+    }
+}
```

Comment:
Perhaps add a newline at the end of the file? I think this was mentioned during lecture that it will be easier to work with Git this way.